pkg/client/unversioned/discovery_client.go

```diff
@@ -50,8 +58,21 @@ type ServerResourcesInterface interface {
 // DiscoveryClient implements the functions that dicovery server-supported API groups,
 // versions and resources.
 type DiscoveryClient struct {
-	httpClient HTTPClient
-	baseURL    url.URL
+	*RESTClient
+}
+
+// ServerVersion retrieves and parses the server's version.
+func (d *DiscoveryClient) ServerVersion() (*version.Info, error) {
+	body, err := d.Get().AbsPath("/version").Do().Raw()
```

Comment:
Into() requires a runtime Object. Shall we make version.Info a runtime Object?

https://github.com/kubernetes/kubernetes/blob/master/pkg/version/version.go#L24